cosign/payload.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is
+# licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dataclasses
+import json
+
+import oci.model as om
+
+
+CosignSignatureType = "cosign container image signature"
```

Comment:
- I think it is rather uncommon to use CamelCase for identifiers that are not type-/class-names in python (see pep8)
- considering the value is only used once, you might inline it, or consider marking it as "private" (-> `_cosign_signature_type`)
- I think we should make this type-str somewhat conformant to MIME-types (e.g. gardener.vnd/oci/cosign-signature)

Reply:
done.
i first had the signature type set to the exact value what cosign is using as hardcoded string ("cosign container image signature"). but tested with your suggestion and it also worked :)